packages/core/admin/admin/src/components/MainNav/tests/NavUser.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { screen, render } from '@tests/utils';
+
+import { NavUser } from '../NavUser';
+
+describe('NavUser', () => {
+  it('shows the initials of the user', async () => {
+    render(<NavUser initials="JD">John Doe</NavUser>);
+    expect(screen.getByText('JD')).toBeInTheDocument();
+  });
+
+  it('contains the user name', async () => {
+    render(<NavUser initials="JD">John Doe</NavUser>);
+    const userName = screen.getByText('John Doe');
+    expect(userName).toBeInTheDocument();
+  });
```

Comment:
for these you should be able to do `getByRole("button", {name: NAME })`, if you prefer.

Reply:
I tried but it doesn't work, but in NavUser there is just one button

Reply:
interesting................ note to self.